### ISSUE TITLE
Crash with segementation fault when launching on host docker network

### ISSUE THREAD

**Issue by jonstahl** [justifiably/docker-musicip#4]
Greetings, and thanks for this really great package — it's been the keystone of my Logitech Music Server system for many months! 

I've been running LMS on my Synology NAS, using the community-maintained LMS package and your MusicIP docker image. However, the coming Synology DSM 7 release is going to break the native LMS package, so I've been experimenting with moving towards a fully-dockerized architecture.

Running docker versions of LMS and MusicIP on the standard bridge network appears to mean that LMS can't find MusicIP, because the LMS MusicIP plugin has a bunch of hard coded calls to "localhost." So I have been attempting to launch both LMS and MusicIP containers connected to my docker host network, being careful to avoid port conflicts. 

When I launch a clean MusicIP container on the host network, it immediately crashes with only a generic "segmentation fault" error in the logs. Same container, same settings, works fine connected to the bridge network. (I am using the Synology docker GUI, rather than docker CLI commands.)

**Comment by Bodge-IT** [comment]
Hi Jon,
I have this combo running with the default musicip image "musicip:latest".
I load musicip and lms in same dockercompose file with lms set to run in "host" mode for networking.

Works no problem.  Not sure if you have docker-compose on Synology but think you should be able to link them manually too.
Pretty sure I tried this image and got the same issue you did.

**Comment by jonstahl** [comment]
Thanks for the tip!

**Comment by justifiably** [comment]
Hi both, glad you are getting some use from this package!  There is actually a patch I made for my dockerised version of LMS to deal with this (and so avoid needing host network mode), I added a MusicIP host name option which I've used for a while.  Unfortunately Michael H was reluctant to merge it into the main release without also adding ways to change paths, and then I realised that some plugins may also hardcode localhost (esp DynamicMix).  My patch has rusted meanwhile.  So I think as @Bodge-IT says, host mode networking or conjuring some port forwarding between containers is best workaround.

See  <https://github.com/Logitech/slimserver/pull/268> and [lms-musicmagic-host.patch](https://github.com/justifiably/docker-logitechmediaserver/blob/master/lms-musicmagic-host.patch)
